src/GameLogic/PlayerActions/ItemConsumeActions/ItemUpgradeConsumeHandlerPlugIn.cs

```diff
@@ -88,21 +88,28 @@ protected virtual bool TryUpgradeItemOption(Item item)
             return false;
         }
 
-        var itemOption = item.ItemOptions.First(o => o.ItemOption?.OptionType == this.Configuration.OptionType);
-        var increasableOption = itemOption.ItemOption;
-        var higherOptionPossible = increasableOption?.LevelDependentOptions.Any(o => o.Level > itemOption.Level && o.RequiredItemLevel <= item.Level) ?? false;
+        var itemOptions = item.ItemOptions.Where(o => o.ItemOption?.OptionType == this.Configuration.OptionType);
+        var increasableOption = itemOptions.First().ItemOption;
+        var higherOptionPossible = increasableOption?.LevelDependentOptions.Any(o => o.Level > itemOptions.First().Level && o.RequiredItemLevel <= item.Level) ?? false;
```

Comment:
More of a theoretical issue: What if there are two options and one can be increased, but the other can not?